### PR TITLE
Script to search LinkedIn

### PR DESCRIPTION
**Pull request by gniting**
## Description
Script to search LinkedIn

## Type of change
- [x] New script command

## Screenshot
image

## Dependencies / Requirements
None

## Checklist
- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/web-searches/search-linkedin.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Search LinkedIn
+# @raycast.mode silent
+# @raycast.packageName Web Searches
+
+# Optional parameters:
+# @raycast.icon https://s3-us-west-2.amazonaws.com/iconduck.prod/assets.00/asstqik48oc9
```
Comment (unnamedd) [inline commands/web-searches/search-linkedin.sh:10]: Would you please add into this PR the LinkedIn logo? Even if the url works well, we used to push the image file to the `images` folder.

File: commands/web-searches/search-linkedin.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+# @raycast.argument1 { "type": "text", "placeholder": "Search for?" }
```
Comment (unnamedd) [inline commands/web-searches/search-linkedin.sh:11]: As you in the line 18 are doing the replacement from "` `" (space) to `%20`, I suggest you to use one more property in the argument: `percentEncoded`. With that property, Raycast will replace the spaces to `%20` automatically for you.
```suggestion
# @raycast.argument1 { "type": "text", "placeholder": "Search for?", "percentEncoded": true}
```

File: commands/web-searches/search-linkedin.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+# Documentation:
+# @raycast.author Nitin Gupta
+# @raycast.authorURL https://twitter.com/gniting
+# @raycast.description Search LinkedIn
+
+open "https://www.linkedin.com/search/results/all/?keywords=${1// /%20}"
```
Comment (unnamedd) [inline commands/web-searches/search-linkedin.sh:18]: And here, you just need to use the parameter itself:
```suggestion
open "https://www.linkedin.com/search/results/all/?keywords=${1}"
```

**Review by unnamedd — CHANGES_REQUESTED** [review]
Thanks for your contribution @gniting!

File: commands/web-searches/search-linkedin.sh (hunk 2 above)
Comment (gniting) [reply commands/web-searches/search-linkedin.sh:11]: thanks for the hint.

File: commands/web-searches/search-linkedin.sh (hunk 3 above)
Comment (gniting) [reply commands/web-searches/search-linkedin.sh:18]: 👍🏼

File: commands/web-searches/search-linkedin.sh (hunk 1 above)
Comment (gniting) [reply commands/web-searches/search-linkedin.sh]: added `linkedin.png` to the `images` folder

**Review by unnamedd — APPROVED** [review]

**Pull request merged**